### PR TITLE
Add role support to russianroulette command

### PR DESCRIPTION
**Pull request by Miroito**

### CODE REVIEW

File: src/modules/commands/discord/russianroulette.ts [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+import { sleep } from '@app/utils/async'
 
-export default new Command({
+export default new Command<discord.Message>({
   keyword: 'russianroulette',
   description: 'kill yourself for fun purposes',
   help: 'Usage: `{prefix}russianroulette` - kill yourself for fun purposes',
-  callback: ({ args, reply, author }) => {
-    if (args.length > 0) return
+  callback: async ({ author, reply, raw }) => {
```
Comment (vostrnad) [inline src/modules/commands/discord/russianroulette.ts:10]: Don't forget to check the number of arguments (return early if > 0).

File: src/modules/commands/discord/russianroulette.ts [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
     const rounds = 6
     const rolled = randomBigInt(BigInt(rounds)) + BigInt(1)
+    const Deadrole = raw.guild?.roles.cache.find((role) => role.name === 'Dead')
+
+    if (!(Deadrole instanceof discord.Role)) {
+      return reply('The Dead role is not defined')
```
Comment (vostrnad) [inline src/modules/commands/discord/russianroulette.ts:16]: Missing period at the end.

File: src/modules/commands/discord/russianroulette.ts [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
+export default new Command<discord.Message>({
   keyword: 'russianroulette',
   description: 'kill yourself for fun purposes',
   help: 'Usage: `{prefix}russianroulette` - kill yourself for fun purposes',
-  callback: ({ args, reply, author }) => {
-    if (args.length > 0) return
+  callback: async ({ author, reply, raw }) => {
     const rounds = 6
     const rolled = randomBigInt(BigInt(rounds)) + BigInt(1)
+    const Deadrole = raw.guild?.roles.cache.find((role) => role.name === 'Dead')
```
Comment (vostrnad) [inline src/modules/commands/discord/russianroulette.ts:13]: Variables are in camelCase by convention, please rename to "deadRole".

**Review by vostrnad — CHANGES_REQUESTED** [review]

File: src/modules/commands/discord/russianroulette.ts (hunk 1 above)
Comment (Miroito) [reply src/modules/commands/discord/russianroulette.ts]: done

File: src/modules/commands/discord/russianroulette.ts (hunk 2 above)
Comment (Miroito) [reply src/modules/commands/discord/russianroulette.ts]: done

File: src/modules/commands/discord/russianroulette.ts (hunk 3 above)
Comment (Miroito) [reply src/modules/commands/discord/russianroulette.ts]: done

**Review by vostrnad — APPROVED** [review]

**Pull request merged**